api/user/schema.py

```diff
@@ -32,36 +33,52 @@ class Arguments:
     user = graphene.Field(User)
 
     def mutate(self, info, **kwargs):
-        user = UserModel(**kwargs)
-        if not verify_email(user.email):
-            raise GraphQLError("This email is not allowed")
-        payload = {
-            'model': UserModel, 'field': 'email', 'value':  kwargs['email']
-        }
-        with SaveContextManager(user, 'User email', payload):
-            notification_settings = NotificationModel(user_id=user.id)
-            notification_settings.save()
-            return CreateUser(user=user)
+        try:
+            user = UserModel(**kwargs)
+            if not verify_email(user.email):
+                raise GraphQLError("This email is not allowed")
+            payload = {
+                'model': UserModel, 'field': 'email', 'value':  kwargs['email']
+            }
+            query = User.get_query(info)
+            result = query.filter(UserModel.state == "active")
+            user_name = result.filter(
```

Comment:
kindly remove this code because its redundant with the presence of the context manager